geomesa-blobstore/geomesa-blobstore-api/src/main/scala/org/locationtech/geomesa/blob/api/GeoMesaPersistentBlobStoreServlet.scala

```diff
@@ -0,0 +1,84 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.blob.api
+
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreFactory}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore
+import org.locationtech.geomesa.web.core.GeoMesaBaseDataStoreServlet
+import org.scalatra.{Ok, BadRequest}
+
+import scala.collection.JavaConversions._
+
+trait GeoMesaPersistentBlobStoreServlet extends GeoMesaBaseDataStoreServlet {
+
+  @volatile var abs: AccumuloBlobStore = null
```

Comment:
so you're persisting aliases now, but you aren't initializing the blobstore on startup, right? and the blob store isn't associated with any particular alias, it is just the last alias registered? seems like the workflow is still a little confused.